server/http/protolet/protolet.go

```diff
@@ -9,10 +9,16 @@ import (
 	"net/http"
 	"reflect"
 
+	ctxpb "github.com/buildbuddy-io/buildbuddy/proto/context"
```

Comment:
nit: style thing but we typically have named imports below unnamed ones, like this:

import (
 "builtins"

 "package/lib_imports"

 foo "named_imports"
)

Reply:
Good to know, done